app/api/auth.py

```diff
@@ -23,6 +24,11 @@
 from app.models.user import User
 
 auth_routes = Blueprint('auth', __name__, url_prefix='/v1/auth')
+with app.app_context():
+     db.init_app(app)
```

Comment:
I'm not sure this is a good idea

Reply:
I guess until initializing the app under a manual context is ok(The application context is created and destroyed as necessary.), as it's working out of the application context.

Reply:
It only initializes the database under a manual context. Um, is this wrong?

Reply:
It should not be needed